src/backend/common/models/event.py

```diff
@@ -720,7 +746,15 @@ def normalized_name(self) -> str:
     @property
     def first_api_code(self) -> str:
         if self.first_code is None:
-            return self.event_short.upper()
+            if self.year == 2023:
```

Comment:
There are some subtle consequences of doing this. I'm going to revert this back.